Pipfile

```diff
@@ -4,6 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+atomicwrites = "*"
```

Comment:
This is required for windows tests. `pytest` requires it but it is not installed by default.